packages/interface-internal/src/address-manager/index.ts

```diff
@@ -41,6 +74,11 @@ export interface AddressManager {
    */
   getAddresses(): Multiaddr[]
 
+  /**
+   * Return all addresses we know about with metadata
```

Comment:
To me this implies that some addresses will not have metadata and will be omitted.  This may be better:

```suggestion
   * Return all known addresses with metadata
```

Reply:
That's true, agreed

Reply:
Updated